swift/ql/test/library-tests/controlflow/graph/Cfg.expected

```diff
@@ -4940,8 +4940,8 @@ cfg.swift:
 #-----|  -> exit deinit
 
 #  378| deinit
-#-----|  -> call to ...
 #-----|  -> TBD (OtherConstructorDeclRefExpr)
+#-----|  -> call to ...
```

Comment:
uh, why is the order changing here?

Reply:
That's a good question to which I don't have an answer right now 😞

Reply:
That's because now we extract more: I just removed the module extraction and diff is gone.

Reply:
well, it's flaky now 😞